README.md

```diff
@@ -87,15 +87,23 @@ For building the driver `docker` and `GO` should be installed on the system
 
    Create an image pull secret in your cluster
 
-   1. Review and retrieve the following values for your image pull secret.
+   1. ibmcloud login to the target region
+
+   2. Run - ibmcloud cr region-set global
+
+   3. Run - ibmcloud cr login
+
+   4. Run - ibmcloud ks cluster config --cluster \<cluster-name\> admin
```

Comment:
Do we really require this for open-source community as we are not sure if they will use IBM managed cluster ?

Reply:
Yes. This can be removed

Reply:
Lets keep it we are anyways recommend if use is pushing to the ibm-repo then ibmcloud cli is mandatory.

Reply:
Can we have this section as optional and its upto user?

also lets not add anything related with `ibmcloud ks`

I see you have removed resizer, why we removed ?

Reply:
Resizer is removed because it is not referred in node deployment file.
https://github.com/kubernetes-sigs/ibm-vpc-block-csi-driver/blob/master/deploy/kubernetes/driver/kubernetes/manifests/node-server.yaml

Only controller has the resizer pod